src/buildStore.js

```diff
@@ -42,7 +42,9 @@ const buildStore = (reducers, devOptions = {}) => {
 		applyMiddleware(routerMiddleware(history), apiMiddleware, spawnerMiddleware),
 	);
 
-	const supportedLocales = SUPPORTED_LOCALES || ["en"];
+	const supportedLocales = SUPPORTED_LOCALES || [
+		{ language: "English", cultureIso: "en-US" },
```

Comment:
Do we need to change 'en' to 'en-US'? In other places it still 'en' as default value ('src/reducers/localeFactory.js' , 'src/selectors/locale.js')